packages/pdf-viewer/ui/IconButtons.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import styled from 'styled-components';
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { faPrint, faDownload, IconDefinition } from '@fortawesome/free-solid-svg-icons';
+
+
+const ButtonElement = styled.button<{ hoverColor?: string; size?: number; color?: string }>`
+    padding: 0.2rem 0.7rem;
+    cursor: pointer;
+    border: solid thin transparent;
+    border-radius: 5px;
+    user-select: none;
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    font-size: ${props=> props.size || 1}rem;
+    background: transparent;
+	color: var(--${props=>props.color || 'grey'});
+    &:hover {
+        background: #7676764d;
+        ${props=>props.hoverColor && `color: var(--${props.hoverColor})`}
+    }
+`;
+
+interface ButtonProps {
+    icon?: IconDefinition;
+    onClick?: ()=> void;
+    name?: string;
+    size?: number;
+    color?: string;
+    hoverColor?: string;
```

Comment:
Why are these all optional? You're setting al of them every time so make them required. Required properties are easier to deal with, especially when refactoring code. Optional properties are good mostly for public APIs, but for internal code it's rarely needed and doesn't help much.